.coveragerc

```diff
@@ -1,5 +1,6 @@
 [run]
 omit = ddtrace/vendor/*
+parallel = true
```

Comment:
what does this do?

Reply:
This resolves an issue with coverage, since we're tracking the coverage per test with ITR, and there is another coverage instance running, there are conflicts while trying to persist the coverage file if we don't use this option.